### PR TITLE
Breakout config test for 4x100, 2x100 and 4x10

### PR DESCRIPTION
**Pull request by mastarkey**
OC test for breakout config / delete for 4x10, 4x100, 2x100

Process is as follows

figure out which breakout mode will be configured in a loop.
once breakout mode determined it will configure that breakout mode
configure IPv4 address on the breakout interfaces
Configure the ATE interface
Ping the remote ATE interface over the breakout and verify the ping response

### CODE REVIEW

**Comment by LimeHat** [comment]
Is there a README for this test?

cc @xw-g @cfernanz

File: feature/experimental/platform/tests/breakout_configuration/breakout_base_test.go [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+		t.Errorf("Breakout speed configured: got %v, want %v", breakoutSpeedGot, breakoutSpeedWant)
+	}
+}
+
+func verifyDelete(t *testing.T, dut *ondatra.DUTDevice, compname string, schemaValue uint8) {
+
+	if errMsg := testt.CaptureFatal(t, func(t testing.TB) {
+		gnmi.Get(t, dut, gnmi.OC().Component(compname).Port().BreakoutMode().Group(schemaValue).Index().Config()) //catch the error  as it is expected and absorb the panic.
+	}); errMsg != nil {
+		t.Log("Expected failure as this verifies the breakout config is removed")
```
Comment (nraghavendran) [inline feature/experimental/platform/tests/breakout_configuration/breakout_base_test.go:49]: pls consider rephrasing this comment and removing the word "failure" as it is prone to confusion.
also, a genuine error msg will be missed in this case, as we are presuming that errMsg will always contain confirmation that the breakout is successfully deleted.

File: feature/experimental/platform/tests/breakout_configuration/breakout_test.go [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+	top := gosnappi.NewConfig()
+	ports := ate.Ports()
+
+	// order the ports from 1 to 8
+	sort.Slice(ports, func(i, j int) bool {
+		return ports[i].ID() < ports[j].ID()
+	})
+
+	// based on setup 100G breakout is on ports 1 to 4 and 10G is ports 5 to 8
```
Comment (nraghavendran) [inline feature/experimental/platform/tests/breakout_configuration/breakout_test.go:76]: is this written specific to a given device/LC model?

**Review by nraghavendran — DISMISSED** [review]

**Review by AmrNJ — APPROVED** [review]
LGTM

**Comment by mastarkey** [comment]
closing due to lack of updates

**Pull request closed without merging**